analysis/import/SCMLogParserV2/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparserv2/parser/StandardCommitParser.kt

```diff
@@ -0,0 +1,57 @@
+package de.maibornwolff.codecharta.importer.scmlogparserv2.parser
+
+import de.maibornwolff.codecharta.importer.scmlogparserv2.input.Commit
+import de.maibornwolff.codecharta.importer.scmlogparserv2.input.Modification
+import de.maibornwolff.codecharta.importer.scmlogparserv2.input.VersionControlledFile
+
+class StandardCommitParser : CommitParser {
+
+    override fun canParse(commit: Commit): Boolean {
+        return !commit.isEmpty && !commit.isMergeCommit()
+    }
+
+    override fun parse(commit: Commit, versionControlledFilesList: VersionControlledFilesList) {
+        commit.modifications.forEach {
+            if (it.currentFilename.isEmpty()) return@forEach
+
+            val trackName = it.getTrackName()
+
+            when (it.type) {
+
+                Modification.Type.ADD -> {
+                    val file = versionControlledFilesList.get(trackName)
+                    if (file != null && !file.isDeleted() && it.currentFilename == file.filename) {
+                        versionControlledFilesList.get(trackName)!!.registerCommit(commit, it)
+                    } else {
+                        val vcfToBeAddedOrReplaced = versionControlledFilesList.addFileBy(trackName)
+                        it.markInitialAdd()
+                        vcfToBeAddedOrReplaced.registerCommit(commit, it)
+                    }
+                }
+
+                Modification.Type.DELETE -> {
+                    // TODO registerCommit() needed? @Ruben do we want to track deleted and then reverted files as author and commit amount
```

Comment:
Add issue to handle REVERTS. Parse default revert commit message to identify REVERTS in git log.
Add an option to parse REVERTS or not.